lib/src/backend/metadata.dart

```diff
@@ -88,27 +91,30 @@ class Metadata {
   /// [testOn] defaults to [PlatformSelector.all].
   Metadata({PlatformSelector testOn, Timeout timeout, bool skip: false,
           this.verboseTrace: false, this.skipReason,
-          Map<PlatformSelector, Metadata> onPlatform})
+          Map<PlatformSelector, Metadata> onPlatform, List<String> tags})
       : testOn = testOn == null ? PlatformSelector.all : testOn,
         timeout = timeout == null ? const Timeout.factor(1) : timeout,
         skip = skip,
         onPlatform = onPlatform == null
             ? const {}
-            : new UnmodifiableMapView(onPlatform);
+            : new UnmodifiableMapView(onPlatform),
+        this.tags = tags ?? const <String>[];
```

Comment:
Yep. Will fix. BTW, the review moved to https://codereview.chromium.org/1405633004/